### PR TITLE
GPIB-112 Transactions Table

### PR DESCRIPTION
**Pull request by callumjg**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>GPIB v2</title>
```
Comment (bitcoinbrisbane) [inline public/index.html]: Maybe spell it out "Get Paid In Bitcoin"

File: src/apis/gpib.js [hunk 2]
```diff
@@ -1,27 +1,27 @@
-import axios from "axios";
+import axios from 'axios';
```
Comment (bitcoinbrisbane) [inline src/apis/gpib.js]: Don't we use double quote?  Can we add the same prettier?

File: public/index.html (hunk 1 above)
Comment (callumjg) [reply public/index.html]: Done.

**Review by bitcoinbrisbane — COMMENTED** [review]
Confirm the prettier styling.  Lets use the same from admin

File: src/apis/gpib.js (hunk 2 above)
Comment (callumjg) [reply src/apis/gpib.js]: Yep done

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**